app/main/views/suppliers.py

```diff
@@ -60,15 +60,22 @@ def suppliers_list_by_prefix():
 def suppliers_details(supplier_id):
     supplier = data_api_client.get_supplier(supplier_id=supplier_id)["suppliers"]
 
-    first_character_of_supplier_name = supplier["name"][:1]
-    if is_alpha(first_character_of_supplier_name):
-        prefix = process_prefix(prefix=first_character_of_supplier_name, format='template')
-    else:
-        prefix = u"other"
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    live_framework_names = [f['name'] for f in all_frameworks if f['status'] == 'live' and f['framework'] == 'g-cloud']
```

Comment:
We could have parameterised the route by framework-framework to give legit DOS suppliers a page.

Reply:
We could have, and I considered doing that initially (going so far as to implement it (bar tests)), but it does swell the scope of this ticket quite a bit.